brevent/src/main/res/values/strings.xml

```diff
@@ -5,20 +5,20 @@
     <string name="app_name_debug">Nrevent</string>
 
     <string name="fragment_guide_welcome">Welcome</string>
-    <string name="fragment_guide_welcome_message">How do you quit apps?\nTapping Back always means that you have finished your work and want to quit.\nTapping Home will start the launcher and kick this app into background.\n\nBrevent recognizes whether you have opened or closed apps by this way, preventing apps which are in the Brevent list from running in long time when not used. Apps which are wakened maliciously will always be force-stopped.</string>
-    <string name="fragment_guide_welcome_message2">When you tap Back to quit apps, the services of these apps will be stopped; tap Home and they will be kept in the background to complete their tasks; swipe apps away from recents screen (also known as the Overview screen, recent task list, or recent apps) and they will be force-stopped.</string>
+    <string name="fragment_guide_welcome_message">How do you quit apps?\nTapping Back always means that you have finished your work and want to quit.\nTapping Home or Overview will kick this app into background.\n\nBrevent recognizes apps\' states by this way, preventing apps which are in the Brevent list from running in long time.</string>
+    <string name="fragment_guide_welcome_message2">When you tap Back to quit apps, the apps as well as their services will no longer run unless you launch them again; tap Home or Overview, or kick apps into background, they and their services can continue running; swipe apps away from recents screen (also known as the Overview screen, recent task list, or recent apps) and they will be force-stopped. Whenever an app is wakened, as long as there is no UI, Brevent will force-stop it. \n\nThe apps in the brevent list can also push notifications or sync as long as set \"Allow sync\". When an app is set \"Allow sync\", if it has noncancelable notification or top services, it will not be brevented; if it has notification or is running in the background, it will not be force-stopped.</string>
 
     <string name="fragment_guide_permission">Permissions</string>
-    <string name="fragment_guide_permission_message">What do you need to do? \nJust a simple command via ADB with your PC.</string>
-    <string name="fragment_guide_permission_message2">Brevent won\'t work if Android-powered device reboots. For Android O, Brevent won\'t work if you turn off \"USB Debugging\" or change USB options.</string>
+    <string name="fragment_guide_permission_message">What do you need to do? \nJust a simple command via ADB with your PC, making brevent has permission to manage other apps indirectly.</string>
+    <string name="fragment_guide_permission_message2">Brevent won\'t work if Android-powered device reboots. For Android 8 or higher, brevent server will stop running if you turn off \"USB Debugging\" or change USB options.</string>
```

Comment:
For Android 8 or higher -> Since Android 8
brevent server -> Brevent server
stop running -> stop working